release-notes/5.0/preview/api-diff/preview2/.Net/5.0-preview2_System.Text.RegularExpressions.md

```diff
@@ -0,0 +1,15 @@
+# System.Text.RegularExpressions
+
+``` diff
+ namespace System.Text.RegularExpressions {
+     public class Regex : ISerializable {
+-        protected bool UseOptionR();
++        protected internal bool UseOptionR();
+     }
+     public class RegexMatchTimeoutException : TimeoutException, ISerializable {
+-        void System.Runtime.Serialization.ISerializable.GetObjectData(SerializationInfo si, StreamingContext context);
++        void System.Runtime.Serialization.ISerializable.GetObjectData(SerializationInfo info, StreamingContext context);
```

Comment:
Since this is my first time signing off on one of these: I'm thinking this is acceptable for a preview release (as Anirudh confirmed on IM), but would likely be considered a breaking change when we ship. With that in mind, do we do anything special for potentially breaking changes in previews? @eerhardt

Reply:
We try to add breaking-change label to the PRs.  When it comes to go-live releases we will document the breaking changes between the releases.

Reply:
For reference, this change came in here: https://github.com/dotnet/corefx/pull/42132/files#r339741013

The `src` file had it named `info`, this synchronized the `ref` file.

/cc @eiriktsarpalis